libraries/chain/webassembly/eos-vm-oc/compile_monitor.cpp

```diff
@@ -278,6 +278,11 @@ void launch_compile_monitor(int nodeos_fd) {
 
 struct compile_monitor_trampoline {
    void start() {
+      struct sigaction sa;
+      sa.sa_handler = SIG_DFL;
+      sa.sa_flags = SA_NOCLDWAIT;
+      sigaction(SIGCHLD, &sa, nullptr);
```

Comment:
This seems risky.  Have you verified that nodeos (or any other program including the unit tests that might use OC) never waits for any subprocess?

Reply:
I felt it was risky also that is why I had it in the cleos process instead of here.  Your comment above was for me to move it.  I did not see where nodeos was waiting, however.

Reply:
Is it still needed at all?  I thought that cleos would no longer start up OC at all after 9317d70.

Reply:
Good point!  Not really.